lib/ansible/modules/cloud/vmware/vmware_datastore_host.py

```diff
@@ -0,0 +1,328 @@
+#!/usr/bin/python
+
+# Copyright (c) 2017 Ansible Project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: vmware_datastore_host
+short_description: Add/remove datastore on ESXi host
+description:
+- This module can be used to mount/umount on datastore on ESXi host.
+- Currently, this module only support NFS datastore type but it's planned to add vmfs support.
+version_added: '2.5'
+author:
+- Ludovic Rivallain <ludovic.rivallain@gmail.com> @lrivallain
+notes:
+- Tested on vSphere 6.0, 6.5
+requirements:
+- python >= 2.6
+- PyVmomi
+options:
+  datacenter_name:
+    description:
+    - Name of the datacenter to add the host.
+    required: true
+  datastore_name:
+    description:
+    - Name of the datastore to add/remove.
+    required: true
+  datastore_type:
+    description:
+    - Type of the datastore to configure (nfs/vmfs).
+    required: true
+  nfs_server:
+    description:
+    - NFS host serving nfs datastore.
+    - Required for nfs datastore type
+    - Unused for others types
```

Comment:
Add a `.` at the end of the sentences.
(same with the other descriptions)

Reply:
Required if datastore type is "nfs" and state is "present", else unused.
(same on other description)